apps/docs/content/components/accordion/controlled.ts

```diff
@@ -1,7 +1,7 @@
 const App = `import {Accordion, AccordionItem} from "@nextui-org/react";
 
 export default function App() {
-  const [selectedKeys, setSelectedKeys] = React.useState(new Set(["1"]));
+  const [selectedKeys, setSelectedKeys] = React.useState<Selection>(new Set(["1"]));
```

Comment:
```suggestion
  const [selectedKeys, setSelectedKeys] = React.useState(new Set(["1"]));
```

As this file is meant to be used as plain "Javascript" file we cannot add typescript code